orders/src/main/java/local/naught/orders/model/Orders.java

```diff
@@ -0,0 +1,104 @@
+package local.naught.orders.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "orders")
+public class Orders
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long ordnum;
+
+    private double ordamount;
+    private double advanceamount;
+    private String orderdescription;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "custcode",
+            nullable = false)
+    @JsonIgnoreProperties({"orders"})
+    private Customers customer;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "agentcode",
+            nullable = false)
+    @JsonIgnoreProperties({"orders"})
+    private Agents agent;
```

Comment:
Line 29 - Make sure to include the field name "customers" so that the agent object assigned to an order will not print out its list of customers (the order object is assigned to only one customer, so you need to show only that).

Also, since lazy fetching is used for this many-to-one relationship, include the property "hibernateLazyInitializer" as well to prevent it from being printed.